python-lib/ontology_tagging/ontology_tagger.py

```diff
@@ -20,6 +19,7 @@
 from utils.language_support import SPACY_LANGUAGE_LOOKUP
 from utils.language_support import SPACY_LANGUAGE_RULES
 from utils.language_support import SPACY_LANGUAGE_MODELS_LEMMATIZATION
+from utils.language_support import PUNCTUATION_CHARACTERS
```

Comment:
Can you move language support in another module folder than "utils"? Language support is a super important module for the plugin, it would be better placed in a folder called nlp (or equivalent).

Reply:
Nitpick: doing one line per import is very verbose, and not very Pythonic: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
> Can you move language support in another module folder than "utils"? Language support is a super important module for the plugin, it would be better placed in a folder called nlp (or equivalent).

Sure ! And since we have an "nlp" folder now, I've added "nlp_utils" in it too. > 49575f5


> Nitpick: doing one line per import is verbose, and not very Pythonic: https://www.python.org/dev/peps/pep-0008/#imports

Done, for the import "from ... import ..." as we discussed earlier , in commit 219312e

Reply:
One last nitpick: if the module "nlp_utils" is inside "nlp", you can just call it "utils"

Reply:
Sure 8276993